File: 0026_remove-duplicates-from-sorted-array/csharp/main.cs
using System;
using System.Text.Json;

public class Program
{
    public static void Main()
    {
        var lines = Console.In.ReadToEnd().Split('\n');
        int[] nums = JsonSerializer.Deserialize<int[]>(lines[0]);
        var result = new Solution().RemoveDuplicates(nums);
        Console.WriteLine(JsonSerializer.Serialize(result));
    }
}

File: 0026_remove-duplicates-from-sorted-array/rust/main.rs
use std::io::{self, Read};

mod solution;
use solution::Solution;

fn main() {
    let mut input = String::new();
    io::stdin().read_to_string(&mut input).unwrap();
    let data: Vec<&str> = input.lines().collect();
    let mut nums: Vec<i32> = serde_json::from_str(data[0]).unwrap();
    let result = Solution::remove_duplicates(&mut nums);
    println!("{}", serde_json::to_string(&result).unwrap());
}
